### DISCUSSION TITLE
[Feature]: nhost - Hasura + Auth + Storage + Serverless Functions + CLI

### DISCUSSION THREAD

**Discussion by Yurgeman** [coollabsio/coolify#3129] [Service Template Requests]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

[Nhost](https://github.com/nhost/nhost/tree/main/examples/docker-compose) is an open source Firebase alternative with GraphQL, built with the following things in mind:

- Open Source
- GraphQL
- SQL
- Great Developer Experience

### Why should this be worked on?

<img width="1000" alt="nhost-diagram" src="https://user-images.githubusercontent.com/7249668/192007159-d6a84990-b81a-4634-9749-b73b14a4e9d0.png">

### Nhost consists of open source software:

- Database: [PostgreSQL](https://www.postgresql.org/)
- Instant GraphQL API: [Hasura](https://hasura.io/)
- Authentication: [Hasura Auth](https://github.com/nhost/hasura-auth/)
- Storage: [Hasura Storage](https://github.com/nhost/hasura-storage)
- Serverless Functions: Node.js (JavaScript and TypeScript)
- [Nhost CLI](https://docs.nhost.io/reference/cli) for local development

**Comment by FranciscoMendes10866** [comment]
It would be awesome if we could have this stack with just one click  👌In addition to what has been mentioned, MinIo could also be used for blob storage.
